doc/source/security/encryption-functions.rst

```diff
@@ -0,0 +1,431 @@
+.. _encryption-functions:
+
+===============================================================================
+Encryption functions
+===============================================================================
+
+Percona Server for MySQL 8.0.28-20 adds encryption functions and variables to manage the encryption range. The functions may take an algorithm argument. Encryption converts plaintext into ciphertext using a key and an encryption algorithm.
+
+A digest uses plaintext and generates a hash value. This hash value can verify if the plaintext is unmodified. You can also sign or verify on digests to ensure that the original plaintext was not modified. You cannot decrypt the original text from the hash value.
+
+When choosing key lengths, consider the following:
+
+* Encryption strength increases the key size and, also, the key generation time.
```

Comment:
Encryption strength increases "with" the key size ...

Reply:
Added.